src/index.html

```diff
@@ -214,49 +214,70 @@
-                            To improve submission processing time, users are encouraged to prevalidate submitted files using the <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">EVA VCF validation suite</a>; more  information can be found at our <a href="?FAQ#collapseThree">FAQs</a>.</p>
+                        <p>Upload your VCF file(s), associated data file(s) and EVA metadata template to your private EVA FTP location.</p>
                     </div>
 
-                    <h3>Converting genetic variation data to VCF</h3>
+                    <h4><span class="icon icon-functional" data-icon="k" aria-hidden="true"></span> Receive</h4>
+                    <div>
+                      <p>The EVA aims to process submissions within two business days. Accession numbers shall be sent via email to the submitter upon successful archival of the deposited data.</p>
+                    </div>
+
+                    <h3>EVA data requirements</h3>
+                    <h4>EVA accepts submission of genetic variation data based on three criteria:</h4>
                     <div>
+                        <ol>
+                            <li>The genome assembly used is International Nucleotide Sequence Database Collaboration (<a href="http://www.insdc.org/" target="_blank">INSDC</a>) registered, or will be at point of submission</li>
+                            <li>The variation data is described in valid VCF file(s) this can be tested prior to submission using the EVA VCF Validation Suite found <a href="https://github.com/EBIvariation/vcf-validator" target="_blank">here</a></li>
+                            <li>Submitted variants are associated with allele frequency values and/or genotypes and/or the raw materials to calculate allele frequency values internally (i.e. Allele Count AND Allele Number values)</li>
+                        </ol>
                         <p>
-                            We advise submitters to consult the <a href="http://samtools.github.io/hts-specs/VCFv4.3.pdf">VCF specification guidelines</a> when converting data to VCF in order to ensure that file(s) generated are valid, as this is a requirement of EVA submission.
+                            All data valid for EVA submission shall be made available via the <a href="?Study Browser">Study Browser</a>
+                            and will be browsable using both the <a href="?Variant Browser">Variant Browser</a> and the <a href="?API">EVA API</a>.
+                            <a href="http://www.ensembl.org/info/docs/tools/vep/index.html" target="_blank">Variant Effect Predictor</a> annotations shall be available for variants mapped to genome assemblies that are known to <a href="http://www.ensembl.org/index.html" target="_blank">Ensembl</a>.

```

Comment:
@garysaunders This paragraph could be better placed next to the brokering to dbSNP etc, as they are both added-value services that we provide, and not a requirement.